lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -413,27 +413,6 @@ const expectations = [
       },
     },
   },
-  {
-    lhr: {
```

Comment:
as mentioned. we add a new assertion into our `perf/expectations.js` smoketest and now we dont need to deal with the flakiness of chromestatus.